account.go

```diff
@@ -30,6 +31,32 @@ type Account struct {
 	Email             string   `json:"email"`
 	ACL               []string `json:"acls"`
 }
+type AccountBandwidthBase struct {
+	Bandwidth *AccountBandwidth `json:"bandwidth"`
+}
+
+// Bandwidth represents a Vultr account bandwidth
+type AccountBandwidth struct {
+	PreviousMonth         AccountBandwidthPeriod `json:"previous_month"`
+	CurrentMonthToDate    AccountBandwidthPeriod `json:"current_month_to_date"`
+	CurrentMonthProjected AccountBandwidthPeriod `json:"current_month_projected"`
+}
+
+// AccountBandwidthPeriod represents a Vultr account bandwidth period
+type AccountBandwidthPeriod struct {
+	TimestampStart            string  `json:"timestamp_start"`
+	TimestampEnd              string  `json:"timestamp_end"`
+	GbIn                      int     `json:"gb_in"`
```

Comment:
Abbreviations should be uppercase `GBIn`

Reply:
Fixed: https://github.com/vultr/govultr/pull/356/commits/50dc4d1c27b6db2ec38abcfb482d91bb7c828233